src/main/resources/templates/laDigitalAssister/languagePreference.html

```diff
@@ -26,17 +26,17 @@
                   </th:block>
                 </th:block>
 
-                <th:block
-                    th:replace="~{fragments/inputs/select :: select(label=#{language-preference.language-to-speak-label}, inputName='languageSpeak', helpText=#{language-preference.language-to-speak-help}, content=~{::languageToSpeakContent})}">
-                  <th:block th:ref="languageToSpeakContent">
-                    <th:block
-                        th:replace="~{fragments/inputs/selectOption :: selectOption(value='English', optionText=#{language-preference.english})}"/>
-                    <th:block
-                        th:replace="~{fragments/inputs/selectOption :: selectOption(value='Spanish', optionText=#{language-preference.spanish})}"/>
-                    <th:block
-                        th:replace="~{fragments/inputs/selectOption :: selectOption(value='Vietnamese', optionText=#{language-preference.vietnamese})}"/>
-                  </th:block>
-                </th:block>
+<!--                <th:block-->
+<!--                    th:replace="~{fragments/inputs/select :: select(label=#{language-preference.language-to-speak-label}, inputName='languageSpeak', helpText=#{language-preference.language-to-speak-help}, content=~{::languageToSpeakContent})}">-->
+<!--                  <th:block th:ref="languageToSpeakContent">-->
+<!--                    <th:block-->
+<!--                        th:replace="~{fragments/inputs/selectOption :: selectOption(value='English', optionText=#{language-preference.english})}"/>-->
+<!--                    <th:block-->
+<!--                        th:replace="~{fragments/inputs/selectOption :: selectOption(value='Spanish', optionText=#{language-preference.spanish})}"/>-->
+<!--                    <th:block-->
+<!--                        th:replace="~{fragments/inputs/selectOption :: selectOption(value='Vietnamese', optionText=#{language-preference.vietnamese})}"/>-->
+<!--                  </th:block>-->
+<!--                </th:block>-->
```

Comment:
Seems like we can get rid of the corresponding fields in LaDigitalAssister?

Reply:
@sree-cfa - sounds good. I was a bit more conservative with removing because we might need it again but realizing we have the code history anyway